scheduler/scheduler-server/src/test/java/functionaltests/monitor/SchedulerMonitorsHandler.java

```diff
@@ -298,13 +297,21 @@ private void addTaskEvent(SchedulerEvent event, TaskInfo taskInfo) {
      * @param event type to look for
      * @return removed JobEventMonitor, or null if not exists.
      */
-    public Optional<JobEventMonitor> removeJobEvent(JobId id, SchedulerEvent event) {
+    private JobEventMonitor removeJobEvent(JobId id, SchedulerEvent event) {
```

Comment:
Why removing optional ? I think it is more clear that the method could return null if in the prototype there is an optional.

Reply:
Actually I reverted back my commit where I also added Optional...